wfe2/wfe.go

```diff
@@ -46,6 +47,8 @@ const (
 	issuerPath     = "/acme/issuer-cert"
 	buildIDPath    = "/build"
 	rolloverPath   = "/acme/key-change"
+	newOrderPath   = "/acme/new-order"
```

Comment:
Could you add this endpoint to the `TestHTTPMethods` test as a POST only endpoint?